.golangci.yml

```diff
@@ -169,5 +169,6 @@ issues:
         - contextcheck
         - errname
         - gochecknoinits
-        - goerr113
+        - init
```

Comment:
Why is this being added?

Reply:
To workaroud an issue with `gochecknoinits` : https://github.com/golangci/golangci-lint/issues/4697.

Reply:
Perhaps I should comment this 😄

Reply:
Could you add a comment to that effect, to avoid head-scratching if this causes errors once golangci-lint is fixed?

Reply:
just did

Reply:
Hah I hadn’t seen your comment!